frontend/src/components/ExpensePresentation.js

```diff
@@ -1,28 +1,19 @@
 import React from "react";
-import { Table } from "react-bootstrap";
+import "../styles/Expenses.css";
 
 class ExpensePresentation extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      fixed: 2000,
-      irregular: 800,
+      expenses: 2800,
     };
   }
   render() {
     return (
-      <Table>
-        <tbody>
-          <tr>
-            <td>Total Fixed Expense</td>
-            <td>{this.state.fixed} €</td>
-          </tr>
-          <tr>
-            <td>Total Irregular Expense</td>
-            <td>{this.state.irregular} €</td>
-          </tr>
-        </tbody>
-      </Table>
+      <div className="expPres">
```

Comment:
Should be className="ExpensePresentation"

Reply:
Yes, more descriptice class names.

Reply:
Resolved in 98ae7b35ef0c274798064fc2b47af099f5e6cfa2